src/autorest.nodejs.csproj

```diff
@@ -79,5 +79,8 @@
   <ItemGroup>
     <DotnetRazor Remove="vanilla\Templates\ReadmeTemplate.cshtml" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
What does this do?

Reply:
Good catch. I originally thought I would need a PackageJson template specifically for Azure, but I don't think my final implementation needed it. I'll check and remove it if it isn't needed.